### PR TITLE
Fix some document errors

### PR DESCRIPTION
**Pull request by jackgr**

### CODE REVIEW

File: docs/templates/registry.md [hunk 1]
```diff
@@ -241,25 +254,24 @@ $ dm deploy <template-name>:<version>
 ```
 
 To resolve the template reference, `dm` looks for a template version directory
-with the given version in the template directory with the given template name.
+with the given version in the template directory with the given template name in
+the default template registry.
 
-By default, it uses the Kubernetes Template Registry. However, you can set a
-different default using the `--registry` flag:
+The default is the [Kubernetes Template Registry](https://github.com/kubernetes/deployment-manager/tree/master/templates),
```
Comment (vaikas-google) [inline docs/templates/registry.md]: Doesn't this now need to point to application-dm-templates as per your change in the dm default?

Comment (jackgr) [inline docs/templates/registry.md]: Yes. Good catch.

On Sunday, November 29, 2015, vaikas-google <notifications@github.com>
wrote:

> In docs/templates/registry.md
> <https://github.com/kubernetes/deployment-manager/pull/134#discussion_r46100998>
> :
>
> >
> > -By default, it uses the Kubernetes Template Registry. However, you can set a
> > -different default using the `--registry` flag:
> > +The default is the [Kubernetes Template Registry](https://github.com/kubernetes/deployment-manager/tree/master/templates),
>
> Doesn't this now need to point to application-dm-templates as per your
> change in the dm default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/deployment-manager/pull/134/files#r46100998>
> .
>


-- 
Jack Greenfield | Google

**Comment by jackgr** [comment]
Fixed. PTAL.

**Comment by vaikas-google** [comment]
LGTM

**Pull request merged**